tool/src/test/resources/org/datacommons/tool/genmcf/resolution/output/report.json

```diff
@@ -14,11 +14,11 @@
       "counters": {
         "Sanity_MissingOrEmpty_statType": "1",
         "Sanity_MissingOrEmpty_dcid": "1",
-        "Resolution_OrphanLocalReference_parent": "1",
-        "Resolution_DcidAssignmentFailure_Song": "1",
-        "Resolution_DcidAssignmentFailure_StatisticalVariable": "1",
-        "Resolution_DcidAssignmentFailure_MusicAlbum": "1",
-        "Resolution_ReferenceToFailedNode_variableMeasured": "1"
+        "Resolution_OrphanLocalReference_parent": "2",
+        "Resolution_DcidAssignmentFailure_Song": "2",
+        "Resolution_DcidAssignmentFailure_StatisticalVariable": "2",
+        "Resolution_DcidAssignmentFailure_MusicAlbum": "2",
+        "Resolution_ReferenceToFailedNode_variableMeasured": "2"
```

Comment:
Oh wondering why this is increased

Reply:
I also noticed this, looking into it right now.

Reply:
Merging from master had caused an extra round of resolution. Fixed now